README.md

```diff
@@ -158,6 +158,15 @@ This plugin creates device plugin endpoints based on the configurations given in
                 "devices": ["154c", "10ed"],
                 "drivers": ["vfio-pci"]
             }
+        },
+        {
+            "resourceName": "mlnx_sriov_rdma",
+            "isRdma": true,
+            "selectors": {
+                "vendors": ["15b3"],
+                "devices": ["1018"],
+                "drivers": ["mlx5_core"]
```

Comment:
@Mmduh-483 please remove driver from here.
Only vendor and devide should be mentioned.

rdma device are not created by mlx5_core driver either. they are created by mlx5_ib driver and pci device is owned by mlx5_core driver.

Reply:
same as above